api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -54,8 +60,37 @@ def assertProjectData(self, project):
         self.assertIsNotNone(sample.tissue_location)
         self.assertIsNotNone(sample.treatment)
 
-    def assertProjectReadmeContains(self, text, project_zip):
-        self.assertIn(text, project_zip.read("README.md").decode("utf-8"))
+    def assertProjectReadmeContent(self, zip_file, project_ids: List[str]) -> None:
+        def get_updated_content(content: str) -> str:
+            """
+            Replace the placeholders PROJECT_ID_{i} and TEST_TODAYS_DATE in test_data/readmes
+            with the given project_id and today's date respectively for format testing."
+            """
+            content = content.replace(
+                "Generated on: TEST_TODAYS_DATE", f"Generated on: {utils.get_today_string()}"
+            )
+            # Map project_ids to their coressponding placeholders with indecies in readmes
+            placeholders = {
+                f"PROJECT_ID_{i}": project_id for i, project_id in enumerate(project_ids)
+            }
+            for placeholder, project_id in placeholders.items():
+                content = content.replace(placeholder, project_id)
```

Comment:
```suggestion
            for i, project_id in enumerate(project_ids):
                content = content.replace(f"PROJECT_ID_{i}", project_id)
```

Reply:
Updated at c56c5cc34d9b85cc88cfa974adfebc18d3da85d8, thank you David!